README.md

```diff
@@ -31,74 +31,47 @@ The basic command to run a baseline scan would look like:
 
 | Argument | Default | Description |
 | --- | --- | --- |
-| `-h`, `--help` | ==SUPPRESS== | show this help message and exit |
-| `-hf`, `--helpFormatted` | False | Print the --help command in markdown table format |
-| `--configFile` | None | Config File - SOOS yaml file with all the configuration for the DAST Analysis (See https://github.com/soos-io/soos-dast#config-file-definition) |
-| `--clientId` | None | SOOS Client ID - get yours from https://app.soos.io/integrate/sca |
-| `--apiKey` | None | SOOS API Key - get yours from https://app.soos.io/integrate/sca |
-| `--projectName` | None | Project Name - this is what will be displayed in the SOOS app |
-| `--scanMode` | baseline | Scan Mode - Available modes: baseline, fullscan, and apiscan (for more information about scan modes visit https://github.com/soos-io/soos-dast#scan-modes) |
-| `--debug` | False | Enable to show debug messages. |
 | `--ajaxSpider` | None | Ajax Spider - Use the ajax spider in addition to the traditional one. Additional information: https://www.zaproxy.org/docs/desktop/addons/ajax-spider/ |
-| `--rules` | None | Rules file to use to INFO, IGNORE or FAIL warnings |
-| `--contextFile` | None | Context file which will be loaded prior to scanning the target |
-| `--contextUser` | None | Username to use for authenticated scans - must be defined in the given context file |
-| `--fullScanMinutes` | None | Number of minutes for the spider to run |
-| `--apiScanFormat` | None | Target API format: OpenAPI, SOAP, or GraphQL |
-| `--level` | INFO | Log level to show: DEBUG, INFO, WARN, ERROR, CRITICAL |
-| `--integrationName` | None | Integration Name - Intended for internal use only. |
-| `--integrationType` | None | Integration Type - Intended for internal use only. |
-| `--scriptVersion` | None | Script Version - Intended for internal use only. |
+| `--apiKey` | None | SOOS API Key - get yours from https://app.soos.io/integrate/sca |
 | `--appVersion` | None | App Version - Intended for internal use only. |
-| `--authDisplay` | None | Minimum level to show: PASS, IGNORE, INFO, WARN or FAIL |
-| `--authUsername` | None | Username to use in auth apps |
-| `--authPassword` | None | Password to use in auth apps |
+| `--authDelayTime` | 5 | Delay time in seconds to wait for the page to load after performing actions in the form. (Used only on authFormType: wait_for_password and multi_page) |
+| `--authFormType` | simple | simple (all fields are displayed at once), wait_for_password (Password field is displayed only after username is filled), or multi_page (Password field is displayed only after username is filled and submit is clicked) |
 | `--authLoginURL` | None | Login url to use in auth apps |
-| `--authUsernameField` | None | Username input id to use in auth apps |
+| `--authPassword` | None | Password to use in auth apps |
```

Comment:
change all the references to "in auth apps" to "when authentication is required"

Reply:
fixed